massgov/mayflower
# Core/typography doc update

## Description
Changes to reflect the actual font support we have in core

## Related Issue / Ticket
- [JIRA issue]()

## Steps to Test
1. Pull the branch
2. Start core
3. View the updates in the browser under tokens/typography
4. You should see only references to Arabic, Cyrillic, Greek, Hebrew, Khmer, Latin
5. In the file you will see that the  others are not removed, just commented out until we add support
